masonry/src/widget/mod.rs

```diff
@@ -55,15 +54,15 @@ pub use split::Split;
 pub use text_area::TextArea;
 pub use textbox::Textbox;
 pub use variable_label::VariableLabel;
+pub(crate) use widget_arena::WidgetArena;
 pub use widget_mut::WidgetMut;
+pub(crate) use widget_pod::CreateWidget;
 pub use widget_pod::WidgetPod;
 pub use widget_ref::WidgetRef;
-pub use zstack::{Alignment, ChildAlignment, HorizontalAlignment, VerticalAlignment, ZStack};
-
-pub(crate) use widget_arena::WidgetArena;
-pub(crate) use widget_pod::CreateWidget;
 pub(crate) use widget_state::WidgetState;
+pub use zstack::{Alignment, ChildAlignment, HorizontalAlignment, VerticalAlignment, ZStack};
 
+pub use self::image::Image;
 use crate::{Affine, Size};
```

Comment:
The re-export of `Image` is split from the other re-exports and bundled with non-exported imports.

Reply:
We could probably address that by factoring `ObjectFit` out (thus removing the `use crate::` part) and doing `pub use self::` for all the other re-exports.